<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyQtApp</class>
 <widget class="QMainWindow" name="MyQtApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyQtApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_cv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>OpenCV</string>
    </property>
    <widget class="QPushButton" name="pushButton_writeimage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Write image</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_imagename">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_openimage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Open image</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_circleparams">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_resetimage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset image</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_imagesize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_draw_circle_objects">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw circle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_undo_circle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Undo(&amp;z)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_draw_rect_objects">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw rect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_eigen">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Eigen</string>
    </property>
    <widget class="QPushButton" name="pushButton_create">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
